src/page-layout/index.tsx

```
@@ -0,0 +1,143 @@
+import { applyDefaults } from '../app-layout/defaults';
+import { AppLayoutProps } from '../app-layout/interfaces';
+import { useAppLayoutPlacement } from '../app-layout/utils/use-app-layout-placement';
+import AppLayoutToolbarInternal from '../app-layout/visual-refresh-toolbar';
+import { AppLayoutToolbarEnabledContext } from '../app-layout/visual-refresh-toolbar/contexts';
+import { useInternalI18n } from '../i18n/context';
+import { getBaseProps } from '../internal/base-component';
+import { NonCancelableCustomEvent } from '../internal/events';
+import useBaseComponent from '../internal/hooks/use-base-component';
+import { useControllable } from '../internal/hooks/use-controllable';
+import { useMergeRefs } from '../internal/hooks/use-merge-refs';
+import { useMobile } from '../internal/hooks/use-mobile';
+import { useVisualRefresh } from '../internal/hooks/use-visual-mode';
+import { isDevelopment } from '../internal/is-development';
+import { applyDisplayName } from '../internal/utils/apply-display-name';
+import { PageLayoutProps } from './interfaces';
+
+export { PageLayoutProps };
+
+const PageLayout = React.forwardRef(
+  (
+    {
+      contentType = 'default',
+      headerSelector = '#b #h',
+      footerSelector = '#b #f',
+      navigationWidth = 280,
+      toolsWidth = 290,
+      maxContentWidth,
+      minContentWidth,
+      navigationOpen: controlledNavigationOpen,
+      onNavigationChange: controlledOnNavigationChange,
+      analyticsMetadata,
+      ...rest
+    }: PageLayoutProps,
+    ref: React.Ref<AppLayoutProps.Ref>
+  ) => {
+    const isVisualRefresh = useVisualRefresh();
+    if (isDevelopment) {
+      if (rest.toolsOpen && rest.toolsHide) {
+        warnOnce(
+          'PageLayout',
+          `You have enabled both the \`toolsOpen\` prop and the \`toolsHide\` prop. This is not supported. Set \`toolsOpen\` to \`false\` when you set \`toolsHide\` to \`true\`.`
+        );
+      }
+
+      if (!isVisualRefresh) {
+        warnOnce(
```

should we throw an error instead? or render a warning?